mksusecd

```diff
@@ -434,6 +439,16 @@ if(defined $opt_size) {
   die "$opt_size: invalid size\n" unless $image_size;
 }
 
+if ($opt_sign_pass_file && $opt_sign_key ) {
```

Comment:
1) it's `$opt_sign_key_id`
2) it's the wrong logic - it will show an 'ignored' warning even if no `$opt_sign_pass` is used